tdi_python/tdicli.py

```diff
@@ -50,11 +51,15 @@
 ipython_app = None
 
 class CIntfTdi:
+    target_type_cls = TargetType
 
-    def __init__(self, dev_id, table_cls, info_cls):
+    def __init__(self, dev_id, table_cls, info_cls, driver_path, utils_path):
+    #def __init__(self, dev_id, table_cls, info_cls):
         self._dev_id = dev_id
-        self._utils = CDLL(install_directory+'/lib/libtarget_utils.so', mode=RTLD_GLOBAL)
-        self._driver = CDLL(install_directory+'/lib/libdriver.so')
+        self._utils = CDLL(utils_path, mode=RTLD_GLOBAL)
```

Comment:
Why do we need to change this code? you means each target can def its own libtarget_utils.so?

Reply:
It means that the targets can specify their own path to the libraries.